backend/src/TodosDatabase.ts

```diff
@@ -0,0 +1,64 @@
+import { TodoDto } from './dto/todoDto';
+import { ConnectionService } from './app.connectionService';
+
+export default class TodosDatabase {
+  private static connectionService = new ConnectionService();
+
+  static async createTodo(todo: TodoDto) {
+    const newTodoOrder = await this.getNextTodoMaxOrder();
+    console.log(newTodoOrder);
+    const insertTodoQuery = `
+    INSERT INTO "todosTable" (id,title,date,status,todosorder)
+    VALUES ($1,$2,$3,$4,$5)
+    RETURNING *;`;
+    const todoValues = [
+      todo.id,
+      todo.title,
+      todo.date,
+      todo.status,
+      newTodoOrder,
+    ];
+    return this.connectionService.query(insertTodoQuery, todoValues);
+  }
+
+  private static async getNextTodoMaxOrder(): Promise<number> {
+    const maxOrderQuery = `SELECT * FROM "todosTable" ORDER BY todosorder DESC LIMIT 1`;
+    const resultRows = (await this.connectionService.query(maxOrderQuery)).rows;
+    const maxOrder = !resultRows.length
+      ? 0
+      : (resultRows[0]['todosorder'] as number);
+    return maxOrder + 1;
+  }
```

Comment:
here it's a little bit more tricky and debatable, 
`getNextTodoMaxOrder` shouldn't be part of this object since this isn't the responsibility of this class.
you can still (and probably should) have a method for getting the **current** max todo order